src/constants.ts

```diff
@@ -4,6 +4,7 @@ import * as path from 'path';
 import format from 'string-format';
 import { PathOsBased } from './utils/path';
 import { ComponentOrigin } from './consumer/bit-map/component-map';
+process.env.MEMFS_DONT_WARN = 'true'; // suppress fs experimental warnings from memfs
```

Comment:
i'm not sure this is the correct place to put it. 
maybe in the main file (bin/bit.js)?

Reply:
I'd put it in `src/app.ts`.